### PR TITLE
Fix issue #168

### PR DESCRIPTION
**Pull request by coutinju**
File modified :
-models: tooltip.js
Add a color attribute to the tooltip's model.
-views: tooltip.js
Handle the modification of the color attribute in the tooltip's view.

### CODE REVIEW

File: dahu/core/app/scripts/views/workspace/objects/tooltip.js [hunk 1]
```diff
@@ -42,7 +47,8 @@ define([
         },
 
         modelEvents: {
-            'change': 'render'
+            'change': 'render',
+            'change:color': 'changeBackgroundColor'
```
Comment (barraq) [inline dahu/core/app/scripts/views/workspace/objects/tooltip.js]: event name is not formatted as described in https://github.com/dahuapp/dahu/issues/110

File: dahu/core/app/scripts/views/workspace/objects/tooltip.js [hunk 2]
```diff
@@ -34,6 +35,10 @@ define([
 
         className: 'object tooltip ui-widget-content',
 
+        initialize: function() {
+            this.changeBackgroundColor();
```
Comment (barraq) [inline dahu/core/app/scripts/views/workspace/objects/tooltip.js]: I would rather update the template with the default color

File: dahu/core/app/scripts/views/workspace/objects/tooltip.js [hunk 3]
```diff
@@ -64,8 +70,8 @@ define([
             this.model.set('posy', position.y);
         },
 
-        onAppWorkspaceTooltipEdit: function() {
-            console.log("TooltipView clicked");
+        changeBackgroundColor: function() {
+            this.el.style.backgroundColor = this.model.get('color');
```
Comment (barraq) [inline dahu/core/app/scripts/views/workspace/objects/tooltip.js]: I would rather do `this.$el.css("background-color", this.model.get('color'))`

File: dahu/core/app/scripts/views/workspace/objects/tooltip.js (hunk 2 above)
Comment (coutinju) [inline dahu/core/app/scripts/views/workspace/objects/tooltip.js]: But when someone will change the default color in the model, he will have to know that he has to change the css. I still do it ?

Comment (coutinju) [inline dahu/core/app/scripts/views/workspace/objects/tooltip.js]: The default color is in the css file main.scss .tooltip. So when someone will change the default color he will have to change it in the css too. There is only {{{text}}} in the template of tooltips.

Comment (barraq) [inline dahu/core/app/scripts/views/workspace/objects/tooltip.js]: the default color is in the tooltip model if I refer to dahu/core/app/scripts/models/objects/tooltip.js it should not be in the CSS

Comment (coutinju) [inline dahu/core/app/scripts/views/workspace/objects/tooltip.js]: My bad, it was in the css and now I put it in the model.
In my next commit I removed the default color from the css file.

**Pull request closed without merging**